src/aind_metadata_upgrader/data_description_upgrade.py

```diff
@@ -93,7 +93,16 @@ def upgrade_funding(cls, old_funding: Any) -> Optional[Funding]:
         elif (
             type(old_funding) is dict and old_funding.get("funder") is not None and type(old_funding["funder"]) is dict
         ):
-            return Funding.model_validate(old_funding)
+            old_funder = old_funding.get("funder")
+            if Organization().name_map.get(old_funder["name"]) is not None:
```

Comment:
Why do you need to construct the old `Organization`? Seems like you should just be able to pass the string into the lookup table.

Reply:
ah, this is handling the case where the funder is already fine?